css-fonts/styles.css

```diff
@@ -0,0 +1,7 @@
+body, headings {
```

Comment:
This will not work to select the headings. You will need to provide the elements actual name, like `h1`, `h2`, etc